.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "themes/ps-org-theme"]
+	path = themes/ps-org-theme
+	url = https://github.com/eternoendless/ps-org-theme.git
```

Comment:
Please move it to PrestaShop organisation

Reply:
Done